scene/gui/tree.h

```diff
@@ -527,6 +530,9 @@ friend class TreeItem;
 
 	void set_value_evaluator(ValueEvaluator *p_evaluator);
 
+	void set_delayed_text_editor_enabled(bool enabled);
+	bool get_delayed_text_editor_enabled() const;
```

Comment:
@akien-mga  mm yeah i changed the functions according to your suggestions but i didn't understand what reduz said, so i will push my changes